packages/styles/scss/components/masthead/_masthead-l1.scss

```diff
@@ -12,6 +12,14 @@
 $search-transition: cubic-bezier(0.2, 0, 0.38, 0.9);
 $search-transition-timing: 95ms;
 
+@mixin mastheadL1Short {
+  height: 48px;
+}
+
+@mixin eyebrowHidden {
+  display: none;
+}
+
```

Comment:
I thought I'd use mixins here since we have to use the same rule to **media query** + **css modifier classes**. I'm not sure it's the ideal solution thought I wanted to hear thoughts on this.

Reply:
hmm I see what you're doing but it looks weird to me since they're only one line

Reply:
Yeah this was an attempt to use the same rule for both media query and class since it's not possible comma combine the two. But yeah it doesn't feel to good to me either. Maybe re-think the style usage for the sticky nav...